nearlib/dev.js

```diff
@@ -12,7 +12,7 @@ const devAccountName = 'alice.near';
 
 module.exports = {
     getConfig: async function() {
-        return JSON.parse(decodeURIComponent(getCookie('fiddleConfig')));
+        return JSON.parse(decodeURIComponent(getCookie('fiddleConfig'))) || {};
```

Comment:
Is this supposed to handle cases when JSON is malformed or just `getCookie` returning `null`?

Reply:
getCookie returning null. This comes up in running command line tests